apps/gitness/src/framework/context/ThemeContext.tsx

```diff
@@ -77,7 +71,7 @@ export function ThemeProvider({ children, defaultTheme }: ThemeProviderProps) {
     if (!colorSchemeMeta.parentNode) {
       document.head.appendChild(colorSchemeMeta)
     }
-  }, [theme, setTheme, systemMode, isInset])
+  }, [theme, setTheme, systemMode])
 
-  return <UIThemeProvider {...{ theme, setTheme, setInset, isLightTheme, isInset }}>{children}</UIThemeProvider>
+  return <UIThemeProvider {...{ theme, setTheme, isLightTheme }}>{children}</UIThemeProvider>
```

Comment:
I wonder why this way done as a spread instead of just using the props as normal 🤔  
```suggestion
  return <UIThemeProvider theme={theme} setTheme={setTheme} isLightTheme={isLightTheme}>{children}</UIThemeProvider>
```

Reply:
In this particular case, the names of the props and values are the same, so the "spread way" just reduces duplications

Reply:
That's true, but spreads in props are _usually_ used to take a larger thing that's been passed, not as a shorthand, so it's less obvious at a glance. I prefer more code that's super obvious, rather than a few chars less that makes you think at all.

Reply:
Fixed